src/activities/ActivityUsageEntity.js

```diff
@@ -735,4 +735,52 @@ export class ActivityUsageEntity extends Entity {
 
 		return this._entity.getLinkByRel(Rels.Alignments.alignmentsHierarchical).href;
 	}
+
+	/**
+	* @returns {string} URL of the associate-grade API, for managing grade association with the activity usage, using working copy, if present
+	*/
+	associateGradeHref() {
+		if (!this._entity || !this._entity.hasLinkByRel(Rels.Activities.associateGrade)) {
+			return;
+		}
+
+		return this._entity.getLinkByRel(Rels.Activities.associateGrade).href;
+	}
+
+	_canCheckout() {
```

Comment:
Is the underscore intended while the `canCheckin` doesn't have one?

Reply:
Yes, it's because `_canCheckout` is only used within this file, whereas `canCheckin` will be used directly by activities.